### DISCUSSION TITLE
TeamSpeak

### DISCUSSION THREAD

**Discussion by aybasaran** [coollabsio/coolify#3863] [Service Template Requests]
https://www.teamspeak.com/en/
![logo_inverse](https://github.com/user-attachments/assets/a5025585-a6cd-48af-81a9-17eb1c58038a)

**Comment by GardoTL** [comment]
🆙

**Comment by LucaNerlich** [comment]
https://github.com/teamspeak/teamspeak6-server

**Comment by LucaNerlich** [comment]
Ok,
the setup is super easy.

1. Create volume / folder on local file system
    - e.g. `/mnt/data-volume-1/teamspeak`
3. `chown -R 9987:9987 /mnt/data-volume-1/teamspeak`
3. `chmod -R 750 /mnt/data-volume-1/teamspeak`
4. Add the below docker-compose as a new resource -> via empty docker compose
5. Set the domain for your server via the coolify services settings section
6. Deploy it

```docker-compose
services:
  teamspeak:
    image: 'teamspeaksystems/teamspeak6-server:latest'
    container_name: teamspeak-server
    restart: unless-stopped
    ports:
      - '9987:9987/udp'
      - '30033:30033/tcp'
    environment:
      - TSSERVER_LICENSE_ACCEPTED=accept
    volumes:
      - '/mnt/data-volume-1/teamspeak:/var/tsserver'
```

**Comment by emreyvz** [comment]
If someone need it: 

install [coolify-mcp](https://github.com/StuMason/coolify-mcp) to any of LLMs that has mcp support. Then prompt it "create me a teamspeak server"
